### PR TITLE
Verify smart contracts with truffle verify

### PR DESCRIPTION
**Pull request by Julian-dev28**
Update for users to verify smart contracts with truffle verify on Avalanche

### CODE REVIEW

File: build/tutorials/smart-contracts/verify-smart-contracts-with-truffle-verify.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+<br>
+
+You should see the txn activity in your terminal
+<img width="793" alt="step1" src="https://user-images.githubusercontent.com/73849597/142695096-4dcbfb58-cc9d-46ba-b146-67ea5f46048f.png">
+<img width="899" alt="step2" src="https://user-images.githubusercontent.com/73849597/142695128-5c4adc0d-78be-423e-ac8e-4a77c81a5d5a.png">
+<img width="794" alt="step3" src="https://user-images.githubusercontent.com/73849597/142695152-bbd7551a-9624-4dca-9649-17fa93a98ebc.png">
+
+<br>
+
+# Verify Smart Contracts with truffle verify
```
Comment (yulin-dong) [inline build/tutorials/smart-contracts/verify-smart-contracts-with-truffle-verify.md:156]: Is this the same title as the top one?

**Review by yulin-dong — CHANGES_REQUESTED** [review]

**Comment by yulin-dong** [comment]
Anything we need to do with https://docs.avax.network/build/tutorials/smart-contracts/verify-smart-contracts-with-sourcify-truffle? Is https://docs.avax.network/build/tutorials/smart-contracts/verify-smart-contracts-with-sourcify-truffle still up-to-date and applicable?

File: build/tutorials/smart-contracts/verify-smart-contracts-with-truffle-verify.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
+# Verifying Samrt Contracts with Truffle Verify
+
+_This tutorial includes items from the truffle [quickstart docs](https://www.trufflesuite.com/docs/truffle/quickstart)_
+_Inspired by [truffle verify docs](https://www.npmjs.com/package/truffle-plugin-verify)_
+
+
+## Create a project
```
Comment (yulin-dong) [inline build/tutorials/smart-contracts/verify-smart-contracts-with-truffle-verify.md:7]: I re-organized the heading, please verify they are correct.

File: build/tutorials/smart-contracts/verify-smart-contracts-with-truffle-verify.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+
+
+You should see the txn activity in your terminal
+<img width="793" alt="step1" src="https://user-images.githubusercontent.com/73849597/142695096-4dcbfb58-cc9d-46ba-b146-67ea5f46048f.png">
+<img width="899" alt="step2" src="https://user-images.githubusercontent.com/73849597/142695128-5c4adc0d-78be-423e-ac8e-4a77c81a5d5a.png">
+<img width="794" alt="step3" src="https://user-images.githubusercontent.com/73849597/142695152-bbd7551a-9624-4dca-9649-17fa93a98ebc.png">
+
+
+
+## Verify Smart Contracts with truffle verify
```
Comment (yulin-dong) [inline build/tutorials/smart-contracts/verify-smart-contracts-with-truffle-verify.md:153]: Is this heading the same as the top one on line 1?

**Review by yulin-dong — CHANGES_REQUESTED** [review]

**Review by Julian-dev28 — COMMENTED** [review]
Good to go

**Review by yulin-dong — APPROVED** [review]

**Pull request merged**